install/etc/services.available/10-db-backup/run

```diff
@@ -27,15 +27,18 @@ else
         print_debug "DB_DUMP_BEGIN is an integer."
         waittime=$(( ${BASH_REMATCH[1]} * 60 ))
```

Comment:
28-29 are off - they should actually be: 

```
        target_time=$(date --date="${today}${DB_DUMP_BEGIN}" +"%s")
        if [[ "$target_time" < "$current_time" ]]; then
            target_time=$(($target_time + 24*60*60))
        fi
        waittime=$(($target_time - $current_time))
```

Reply:
Thank you.